scripts/user_retirement/setup.cfg

```diff
@@ -0,0 +1,12 @@
+[options]
+packages = utils.thirdparty_apis
+
+[options.entry_points]
+# `retirement_driver` is the namespace chosen by our plugin manager
+# this driver should register itself to `retirement_driver`
+retirement_driver =
```

Comment:
Any other python packages can be installed that have a setup.cfg like this to register their plugins with the system. They can still be turned on and off by adding or removing the steps in the retirement configuration file.